package/gluon-state-check/files/usr/lib/micron.d/gluon-state

```diff
@@ -0,0 +1 @@
+* * * * * /usr/sbin/gluon-state
```

Comment:
I would suggest to rename the binary to `gluon-state-check` as well.

Reply:
resolved in ce76c7b69d1d10ad39583ffb2dcfc5defb036ce4